tests/e2e/support/objects/app-admin-settings/groups/actions.ts

```diff
@@ -17,3 +18,15 @@ export const createGroup = async (args: { page: Page; key: string }): Promise<vo
     page.locator(actionConfirmButton).click()
   ])
```

Comment:
How can I get response body here and set uuid group to Group. Maybe you have ideas?

Reply:
I'm not entirely sure but but this could work?
```
page.locator(actionConfirmButton).click()
const resp = page.waitForResponse(
    (resp) =>
        resp.url().endsWith('groups') && resp.status() === 200 && resp.request().method() === 'POST'
)
const body = await resp.body()
```

Reply:
You could do this
```js
const [response] = await Promise.all([
    page.waitForResponse(
      (resp) =>
        resp.url().endsWith('groups') && resp.status() === 200 && resp.request().method() === 'POST'
    ),
    page.locator(actionConfirmButton).click()
  ])
const group = await response.json()
// get the group id from the response
console.log(group.id)
```